charts/rancher-logging/0.2.3/charts/fluentd/charts/fluentd-windows/templates/daemonset.yaml

```diff
@@ -45,6 +45,15 @@ spec:
         volumeMounts:
         - mountPath: /var/lib/rancher
           name: rancher
+      - name: init-{{ template "fluentd.fullname" . }}-02
+        image:  {{ template "system_default_registry" . }}{{ .Values.image.repository }}:{{ .Values.image.tag }}
+        command:
+          - 'powershell'
+          - '-command'
+          - 'New-Item -ItemType Directory -Force -Path /var/lib/rancher/rke/log'
+        volumeMounts:
+        - mountPath: /var/lib/rancher
+          name: rancher
```

Comment:
Want to confirm that we want to mount same volume in both containers.

Reply:
Yes, we do